backend/src/main/java/com/group/backend/cron/ApiConsumer.java

```diff
@@ -0,0 +1,13 @@
+package com.group.backend.cron;
+
+import org.json.JSONObject;
+import org.springframework.web.client.RestTemplate;
+
+public class ApiConsumer {
+	public static JSONObject getData(String url) {
```

Comment:
Pq tanto recuo? '-'

Reply:
Pq precisamos que seja um static method? Pergunta sincera.

Reply:
> Pq tanto recuo? '-'

Vou tentar configurar um Lint, como comentado em sala.

> Pq precisamos que seja um static method? Pergunta sincera.

Como a execução do consumo da API é o mesmo em todas as API, prefiri fazer o ApiConsumer uma classe de "utilidade". No lugar de precisar instanciar um ApiConsumer para cada API que a gente tiver, eu só adquiro o valor dado uma URL qualquer.

Mais ou menos como `Math.sqrt(value);` no lugar de um `Number value = 2; value.sqrt();`

Uma referencia boa:
https://medium.com/@jbhyunikim/unleashing-the-java-dependency-dilemma-static-classes-vs-dependency-injection-showdown-b2a053bc841

É possivel consiguir fazer o mesmo por dependency injection, mas eu não sei muito bem como funciona em Java

Reply:
perfeito!